cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -259,17 +260,17 @@ trait CassandraChunkSource extends RawChunkSource with StrictLogging {
                         columnIDs: Seq[Types.ColumnId],
                         partMethod: PartitionScanMethod,
                         chunkMethod: ChunkScanMethod = AllChunkScan): Observable[RawPartData] = {
-    val indexTable = getOrCreateIndexTable(dataset.ref)
+    val ingestionTable = getOrCreateIngestionTable(dataset.ref)
     logger.debug(s"Scanning partitions for ${dataset.ref} with method $partMethod...")
-    val (filters, indexRecords) = partMethod match {
+    val (filters, infoRecords) = partMethod match {
       case SinglePartitionScan(partition, _) =>
-        (Nil, indexTable.getIndices(partition))
+        (Nil, ingestionTable.getInfos(partition))
```

Comment:
Can we include a couple of Kamon traces to measure the breakdown of how much time we spend in fetching infos vs chunks.

Reply:
I don't know how to measure this time with Cassandra, Futures, Observables, etc. Is there an example of this somewhere?

Reply:
Search for  `val span = Kamon.buildSpan("write-chunkset").start()` in CassandraColumnStore.scala

Reply:
I think this work belongs in a separate PR.